elasticapm/traces.py

```diff
@@ -159,6 +170,16 @@ def __init__(self, transaction, name, span_type, context=None, leaf=False):
         self.duration = None
         self.parent = None
         self.frames = None
+        if not span_subtype and "." in span_type:
```

Comment:
I think we should allow an explicit empty subtype, like:

```suggestion
        if span_subtype is None and "." in span_type:
```